WorksheetB

```diff
@@ -0,0 +1,39 @@
+# Fallout word guessing game
+import random 
+correct word =  pick random word from (selection words of all 5 char long)
+position one character = corrrect answer 1
+position two character = corrrect answer 2
+position three character = corrrect answer 3
+position four character = corrrect answer 4
+position five character = corrrect answer 5
+likeness = 0
+
+incorrect guesses = 0
+print ('What's your guess?')
+guess = input()
+
+
+while incorrect guesses<5 
+	If Length of guess != 5:
+		print ('What you playing at?')
+		add one to incorrect guesses
+	
+	elif length of guess == 5:
+		position 1 of input == correct answer 1:
```

Comment:
These should be formatted as if statements.

Also this is repetitive and would be better formatted as a loop, e.g.
```
for i = 1, ..., n
   if position i of input == position i of correct answer
      add 1 to likeness
```